base/ltcmdhooks.dtx

```diff
@@ -75,9 +87,12 @@
 %    \end{description}
 %
 %    The hooks are not physically present before
-%    \verb=\begin{document}= (i.e., using a command in the preamble
-%    will never execute them) and if nobody has declared any code for
-%    them, then they are not added to the command code ever.  For
+%    \verb=\begin{document}=\footnote{More specifically, they are
+%    inserted in the commands after the \hook{begindocument} hook, so
+%    they are also not present while \LaTeX{} is reading the
+%    \texttt{.aux} file.} (i.e., using a command in the preamble
```

Comment:
haven't I read a discussion that this was changed so that they are available during .aux reading? If so some of the later documentation would need to change too.

Reply:
I think you skipped a word while reading: they are _not_ present when reading the `.aux`. Thus my comment to Ulrike: https://github.com/latex3/latex2e/pull/1068#discussion_r1200456488

Reply:
Right I misread.